pkg/constant/constant.go

```diff
@@ -12,5 +12,11 @@ const (
 	CmdMigrateTssFunds = "cmd_migrate_tss_funds"
 
 	// BTCWithdrawalDustAmount is the minimum satoshis that can be withdrawn from zEVM to avoid outbound dust output
+	// The Bitcoin protocol sets a minimum output value to 546 satoshis (dust limit) but we set it to 1000 satoshis
 	BTCWithdrawalDustAmount = 1000
+
+	// SolanaWalletRentExempt is the minimum balance for a Solana wallet account to become rent exempt
+	// The Solana protocol sets minimum rent exempt to 890880 lamports but we set it to 1_000_000 lamports (0.001 SOL)
+	// The number 890880 comes from CLI command `solana rent 0` and has been verified on devnet gateway program
+	SolanaWalletRentExempt = 1_000_000
```

Comment:
This value is the same across all networks?

Reply:
Yeah.

Reply:
@swift1337 right now, yeah. Totally 5 shared constants by `zetacore` and `zetaclient` ATM. Other non-shared constants are defined in chain's own package.